admin_app_2/src/components/ContentCard.tsx

```diff
@@ -42,26 +41,35 @@ const ContentCard = ({
           appStyles.shadow,
         ]}
       >
-        <Layout.FlexBox
-          flexDirection="row"
-          gap={sizes.tinyGap}
-          sx={{ justifyContent: "start", letterSpacing: 2 }}
-        >
-          <Typography variant="overline">#{content_id}</Typography>
-        </Layout.FlexBox>
+        <Typography variant="overline" sx={{ fontFamily: "monospace" }}>
+          #{content_id}
+        </Typography>
         <Typography variant="h6" noWrap={true}>
           {title}
         </Typography>
-        <Layout.Spacer multiplier={0.5} />
         <Typography
           variant="body2"
-          paragraph={true}
           color={appColors.darkGrey}
           sx={appStyles.threeLineEllipsis}
         >
           {text}
         </Typography>
-        <Layout.Spacer multiplier={1} />
+        <Layout.Spacer multiplier={0.5} />
```

Comment:
Also realize that there is space between the edit icon and the world "EDIT". I'll fix that in the next PR.

Reply:
Added "Last updated on" and removed italics. Added a space between the edit icon and "EDIT" text too. 

Will keep the datetime below the blurb since it's metadata and it makes more sense to me for it come after the content.